src/amcrest/__init__.py

```diff
@@ -30,5 +30,6 @@ def __init__(self, host, port, user,
             protocol=protocol,
             ssl_verify=ssl_verify,
             retries_connection=retries_connection,
-            timeout_protocol=timeout_protocol
+            timeout_protocol=timeout_protocol,
+            channel=channel
```

Comment:
You need to add a similar parameter to `Http.__init__()` in http.py and use it throughout the entire project where it makes sense.